common/app/views/fragments/mostPopularPlaceholder.scala.html

```diff
@@ -13,9 +15,22 @@ <h2 class="fc-container__header__title">
                     <a href="@LinkTo(url)" data-link-name="Most viewed @section">most viewed</a>
                 </h2>
             </div>
-            <div class="fc-container__body fc-container--rolled-up-hide js-popular-trails">
-
+            <div class="fc-container__body fc-container--rolled-up-hide popular-trails">
+                <div class="popular-trails__content js-popular-trails">
+                </div>
+                <div class="popular-trails__mpu">
+                    <div class="fc-slice__popular-mpu js-fc-slice-mpu-candidate fc-slice__item--mpu-candidate">
```

Comment:
Hmm, depends. I was keeping it so unfilled slots could be closed, for example:

The Children's book section doesn't show ads, so at a tablet breakpoint it has a large empty space where the MPU would be:

<img width="761" alt="picture 553" src="https://cloud.githubusercontent.com/assets/8607683/23212937/dad0888a-f900-11e6-95a8-37f0b9ab5117.png">

If we keep `js-fc-slice-mpu-candidate`, then the `fc-slice__item--no-mpu` class will be added by `render-advert.js`, I can use this to close the slot:

<img width="773" alt="picture 552" src="https://cloud.githubusercontent.com/assets/8607683/23213113/98f2f974-f901-11e6-9380-4e5f9d5e21f7.png">

Or would it be better to leave it open?